src/modules/events/components/EventListItem.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import * as React from 'react'
+import { View } from 'react-native'
+import styled from 'styled-components'
+import { EventModel } from '@integreat-app/integreat-api-client'
+import ListItem from '../../../modules/common/components/ListItem'
+import EventPlaceholder1 from '../assets/EventPlaceholder1.jpg'
+import EventPlaceholder2 from '../assets/EventPlaceholder2.jpg'
+import EventPlaceholder3 from '../assets/EventPlaceholder3.jpg'
+
+const WrappedText = styled.Text`
```

Comment:
Can't this be removed if you don't style anything here?

Reply:
Removed it.